README.md

```diff
@@ -1,25 +1,77 @@
 
 | **Due Date**  | **Wednesday, March 3rd at 1:00PM EST**                                 |
 |---------------|----------------------------------------------------------------------------|
-|  **Submission**   | `ros_exercises.zip` on [Gradescope](https://gradescope.com/)|
+|  **Submission**   | `log.npf` and `ros_exercises.zip` on [Gradescope](https://gradescope.com/) (listed as two separate assignments)|
 
 # Lab 1C: Intro to ROS
```

Comment:
can you add a subsection ##Introduction as in https://github.com/mit-rss/intro_to_git?